hw-6/microsoft/css/style.scss

```diff
@@ -0,0 +1,340 @@
+/* Переменные */
+
+$grey: #262626;
+$background: #F3F3F3;
+
+/* Шаблоны */
+
+%padding {
+    padding: 0 64px;  
+}
+
+%nav-link {
+    margin: 0 10px;
+    text-decoration: none;
+    color: $grey;
+    font-size: 13px;
+}
+
+%nav-link-icons {
+    width: 13px;
+    height: 13px;
+    padding: 10px;
+    text-decoration: none;
+    color: $grey;
+    font-size: 13px;
+}
+
+%nav-link-hover {
+    border-bottom: 2px solid $grey;
+    padding-bottom: 3px;
+}
+
+%footer-link {
+    font-size: 11px;
+    line-height: 16px;
+    color: #616161;
+    text-decoration: none;
+}
+
+/* Миксины */
+
+@mixin font($fs, $fw, $lh) {
+    font-size: $fs;
+    font-weight: $fw;
+    line-height: $lh;
+}
+
+/* Всем шрифт!!!!!!!!!! */
+
+@font-face{
+    font-family: 'Segoe';
+    src: url('../fonts/segoe.ttf') format('truetype');
+}
+
+body * {
+    font-family: 'Segoe', Tahoma, sans-serif;
+    font-weight: 200;
+}
+
+/* Дополнительные классы типовой стилизации */
+
+.flex-row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.flex-column {
+    display: flex;
+    flex-direction: column;
+}
+
+.rounded {
+    border: 1px solid $grey;
+    border-radius: 50%;
+    margin-left: 10px;
+}
+
+/* Макет хедера сайта */
+
+.header {
+    @extend %padding;
+    justify-content: space-between;
+
+    .header__logo-n-nav {
+        a {
+            .header__logo {
+                width: 107px;
+                padding: 16px 12px 16px 10px;
+            }
+        }
+    }
+
+    .header__nav {
+        padding-left: 10px;
+
+        .header__nav-menu {
+            .header__nav-menu-item {
```

Comment:
`
.header {
  &__nav-menu {...}
  &__nav-menu-item-link {...}
  &__nav-menu-item-link:hover {...}
}
`

for example

Reply:
Ok